updated the logo and paragraph

SAM-Aryan opened the pull request:
1. updated the logo on the hero section and in the public accessible favicon
2. made the about section a bit shorter
3. connected the form with the register button
4. changed the name "ideaoryx" to "IdeaOryx"
@dat-adi

Commit:
updated the logo and paragraph

dat-adi commented on src/Loader.js:
```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import logo from '../src/assets/Ideaoryx.svg'
+import logo from '../src/assets/Ideaoryx.png'
```
Is it possible for you to show the output for this for different viewports?
A screenshot would be cool, because I'm not sure if a `png` would scale as well as an `svg` for this case.

SAM-Aryan replied:
okay i will change png to svg

dat-adi commented on src/components/timeline/MuiTimeline.js:
```diff
@@ -81,7 +81,7 @@ function MuiTimeline() {
               Review
             </Typography>
-            <Typography>Submitted ideas will be evaluated by an expert panel based on several parameters.</Typography>
+            <Typography>Submitted project will be evaluated by an expert panel based on several parameters.</Typography>
```
`project` => `projects`.

SAM-Aryan replied:
on it

dat-adi reviewed (changes requested):
Changes with respect to the name change look good.
There are a few issues with the changes though.

- Check into whether the logo plays well with varying viewport sizes.
- Get rid of either `package-lock.json` or `yarn.lock`, both of them together are unnecessary.

Commit:
made minor changes

- updated the ideaoryx png logo to svg
- changed project to projects in timeline file

SVijayB reviewed (approved):
Changes have been reviewed. Everything looks good.
Approving for a merge.
